jormungandr/src/settings/logging.rs

```diff
@@ -28,6 +31,7 @@ pub enum LogFormat {
 /// Output of the logger.
 pub enum LogOutput {
     Stderr,
+    Gelf,
```

Comment:
seems that if you make `Gelf(String, String)` you same yourself the trouble of extending the generic structure for something specific to the backend

Reply:
The structopt/YAML thing would need to be customized for this, I believe.

This whole enum needs love, but I can improve it later.

Reply:
yeah, I was not sure how the yaml parsing would workout if I did that. I can try.

Reply:
ok nevermind, if it's too complicated we can push this through and revisit later